scripts/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# INSTRUCTIONS:
+# This script will move the pre-push hook from script folder to
+# the .git/hooks folder
+#
+# Run the script from the oppia-android root folder:
+#
+#   bash scripts/setup.sh
+#
+# NOTE: this script should be run once after the initial codebase setup
```

Comment:
I think one of my earlier concerns is still a problem: how do we actually get people to do this if everyone is using Android Studio by default?

Reply:
This is something anyone has to do it by themself after initial codebase setup. 

reference - https://github.com/oppia/oppia-android/pull/2002#issuecomment-707601602

Reply:
Okay, in that case I think this is fine. Vinita's plan is a good one, and addresses my concern. Thanks!